gui/elements/_layer_list.py

```diff
@@ -159,6 +163,9 @@ def append(self, item):
         _check_layer(item, error=True)
 
         self._list.append(item)
+        item.name = item.name + str(self.total)
```

Comment:
I don't like doing this outside of a default-name setting scenario. Not necessarily in this PR but somewhere down the line we should make some function for resolving name conflicts; i.e. if we have a `Layer 1` creating a new layer should result in `Layer 2`, but copy/pasting it should result in `Layer 1 1`.

Reply:
Agreed - this does feel out of place here, and it's not working now appropriately anyway. Dealing with layer naming sounds good, being able to pass in a name during the creation of a layer either as meta-data or a keyword argument would be nice too, but that can be a future PR as well.